src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        // Creates three balls using class Lottery and print information about them in console
+        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
+            Lottery lotteryBall = new Lottery();
```

Comment:
do not create Lottery inside loop
![image](https://user-images.githubusercontent.com/20326858/224543103-2a459414-ed6a-4d36-ac4f-206956e92202.png)